KerbalVR_Mod/KerbalVR/InternalModules/KerbalVR_Button.cs

```diff
@@ -8,118 +8,118 @@
+                    gizmo.transform.SetParent(transform, false);
+                    Utils.SetLayer(gizmo, 20);
+                }
 #endif
-			}
+            }
 
-			cover = gameObject.GetComponent<VRCover>();
-		}
+            cover = gameObject.GetComponent<VRCover>();
+        }
 
-		class VRButtonInteractionListener : MonoBehaviour, IFingertipInteractable
-		{
-			public VRButton buttonModule;
+        class VRButtonInteractionListener : MonoBehaviour, IFingertipInteractable
+        {
+            public VRButton buttonModule;
 
-			// when the fingertip initially makes contact, where is its center along the axis?
-			float initialContactOffset = 0.0f;
-			Vector3 initialLocalPosition; // the button's localPosition at rest
-			bool latched = false;
-			bool latchedCover = false;
+            // when the fingertip initially makes contact, where is its center along the axis?
+            float initialContactOffset = 0.0f;
+            Vector3 initialLocalPosition; // the button's localPosition at rest
+            bool latched = false;
+            bool latchedCover = false;
 
-			public void Awake()
-			{
-				initialLocalPosition = transform.localPosition;
+            public void Awake()
+            {
+                initialLocalPosition = transform.localPosition;
 
 #if PROP_GIZMOS
-				Utils.GetOrAddComponent<ColliderVisualizer>(gameObject);
+                Utils.GetOrAddComponent<ColliderVisualizer>(gameObject);
 #endif
-			}
+            }
 
-			float GetFingertipPosition(Vector3 fingertipCenter)
-			{
-				Vector3 localFingertipPosition = transform.InverseTransformPoint(fingertipCenter);
-				return Vector3.Dot(localFingertipPosition, buttonModule.axis);
-			}
+            float GetFingertipPosition(Vector3 fingertipCenter)
+            {
+                Vector3 localFingertipPosition = transform.InverseTransformPoint(fingertipCenter);
+                return Vector3.Dot(localFingertipPosition, buttonModule.axis);
+            }
 
-			public void OnEnter(Hand hand, Collider buttonCollider, SteamVR_Input_Sources inputSource)
+            public void OnEnter(Hand hand, Collider buttonCollider, SteamVR_Input_Sources inputSource)
             {
-				initialContactOffset = GetFingertipPosition(hand.FingertipPosition);
+                initialContactOffset = GetFingertipPosition(hand.FingertipPosition);
 
-				latchedCover = buttonModule.cover != null && !buttonModule.cover.IsOpen;
+                latchedCover = buttonModule.cover != null && !buttonModule.cover.IsOpen;
 
-				if (!latchedCover)
-				{
-					HapticUtils.Light(inputSource);
-				}
-			}
+                //if (!latchedCover)
+                //{
+                //	HapticUtils.Light(inputSource);
+                //}
+            }
 
-			public void OnExit(Hand hand, Collider buttonCollider, SteamVR_Input_Sources inputSource)
+            public void OnExit(Hand hand, Collider buttonCollider, SteamVR_Input_Sources inputSource)
```

Comment:
Could remove inputSource from these as well; looks like I missed that in a previous change.

Reply:
Fixed in 8141003.